src/Components/HostsSection/HostsSection.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./HostsSection.css";
+import Host from "../Host/Host.js";
+
+export default function HostsSection() {
+    return (
+        <div className="hostsSectionContainer">
+            <h2 className="hostsSectionBanner">Meet the HUGE Crew</h2>
+            <div className="hostsContainer">
+                <Host />
+                <Host />
+                <Host />
+                <Host />
```

Comment:
not sure if this will matter for your design or not but there are currently 5 members of the podcast